evostar2019/angrybirds-evo19.tex

```diff
@@ -122,8 +124,40 @@
 %
 \section{Introduction}
 \label{sec:intro}
-\input{Context}
-\input{Motivation}
+%\input{Context}
+%\input{Motivation}
+
+% Antonio - I have written a text closer to what a paper needs to include. We must be quite concise and explain just what it is required to justify our work. ;D
```

Comment:
Once again, that was already done and revised. We don't need another introduction